src/util-file.c

```diff
@@ -662,6 +662,9 @@ static int FileStoreNoStoreCheck(File *ff)
 static int AppendData(
         const StreamingBufferConfig *sbcfg, File *file, const uint8_t *data, uint32_t data_len)
 {
+    DEBUG_VALIDATE_BUG_ON(
+            data_len > BIT_U32(26)); // 64MiB as a limit per chunk seems already excessive
```

Comment:
This is good because SMB records which an be big are streamed, right ?

Reply:
Yes, they are streamed. So per call size should remain limited.